libindy/indy-api-types/Cargo.toml

```diff
@@ -16,7 +16,7 @@ libc = "0.2.66"
 log = "0.4.8"
 openssl = {version = "0.10", optional = true}
 rust-base58 = {version = "0.0.4", optional = true}
-rusqlite = {version = "0.20", optional = true}  # Make sure rusqlite for android is also bumped with this. Rusqlite for android is at the bottom of this document.
+rusqlite = {version = "0.24.2", optional = true}  # Make sure rusqlite for android is also bumped with this. Rusqlite for android is at the bottom of this document.
```

Comment:
do we need to mix this rusqlite version bump to the PR of indy node / plenum changes?

Reply:
I was getting an error with version 0.20.0 when testing locally; I had to bump the version and update the "match" conditions to get the code to compile and tests to pass (errors in previous comments on this PR)

Reply:
probably your local issues is reasoned by incompatibility of rust version and the rusqlite version. Do you use 1.46.0 rust or smth newer?

Reply:
Yep it was a rust version issue good catch!